gordo_components/model/factories/lstm_autoencoder.py

```diff
@@ -41,16 +44,16 @@ def lstm_model(
         One timestep = current observation in the sample.
         Two timesteps = current observation + previous observation in the sample.
         ...
-    encoding_func: list
+    encoding_dim: tuple
+        Tuple of numbers with the number of neurons in the encoding part.
+    decoding_dim: tuple
+        Tuple of numbers with the number of neurons in the decoding part.
+    encoding_func: tuple
         Activation functions for the encoder part.
-    decoding_func: list
+    decoding_func: tuple
         Activation functions for the decoder part.
     func_output: str
```

Comment:
I know you didn't change this, but just noticed it now; this parameter doesn't exist. Should be renamed to `out_func` I suppose.

Reply:
It seems like `out_func` is used in the derivative functions where one only designates the function to be used and the dimensions are calculated from the inputs. 

Here it seems like `decoding_func` is being used [here](https://github.com/ryanjdillon/gordo-components/blob/dimension_function_error_checking/gordo_components/model/factories/lstm_autoencoder.py#L89-L92), just getting pulled out as `activation`.

```python
 89     # decoding layers
 90     for i, (n_neurons, activation) in enumerate(zip(decoding_dim, decoding_func)):
 91         return_seq = i != len(decoding_dim) - 1  # Don't return sequences on 2nd to last
 92         model.add(LSTM(n_neurons, activation=activation, return_sequences=return_seq))
```